nilearn/plotting/html_surface.py

```diff
@@ -130,6 +192,8 @@ def _fill_html_template(info, embed_js=True):
 def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
                      threshold=None, cmap=cm.cold_hot,
                      black_bg=False, vmax=None, vmin=None, symmetric_cmap=True,
+                     bg_map='auto', bg_on_data=False,
+                     bg_map_rescale=True, darkness=.5,
```

Comment:
Should we rescale by default ? Not sure we should...
(Remember that in most cases, this will be to use curv or sulcal texture)

Reply:
Maybe I'm missing something, but it's actually because sulc and curv maps are not scaled that I set `bg_map_rescale=True`.
Is that what you expected?

```python
In [1]: from nilearn import datasets, plotting, surface

In [2]: fsaverage = datasets.fetch_surf_fsaverage()

In [3]: import nibabel as nib

In [4]: curv_left = nib.load(fsaverage["curv_left"]).darrays[0].data

In [6]: import numpy as np

In [7]: np.min(curv_left), np.max(curv_left)
Out[7]: (-0.40463305, 0.34974468)

In [8]: sulc_left = nib.load(fsaverage["sulc_left"]).darrays[0].data

In [9]: np.min(sulc_left), np.max(sulc_left)
Out[9]: (-1.4937248, 1.8069096)

```

Reply:
OK.